tests/test_results.py

```diff
@@ -0,0 +1,69 @@
+from config import API_KEY, WILDFIRE_SCENARIO_ID, SIMPLESAT_SCENARIO_ID
+from sedaro import SedaroSimulationResult, SedaroAgentResult, SedaroBlockResult, SedaroSeries
+from tempfile import TemporaryDirectory
+from pathlib import Path
+
+
+def test_query_terminated():
+    '''Test querying of a terminated scenario.'''
+    result = SedaroSimulationResult.get_scenario_latest(API_KEY, WILDFIRE_SCENARIO_ID)
```

Comment:
Could we import HOST and pass it into the `get_scenario_latest` methods in this file? That allows for using the config to run the tests locally as well as against the live website. Thank!